tests/vmi_networking_test.go

```diff
@@ -863,3 +836,42 @@ func NewRandomVMIWithInvalidNetworkInterface() *v1.VirtualMachineInstance {
 	vmi.Spec.Domain.Devices.Interfaces[0].Model = "gibberish"
 	return vmi
 }
+
+func createExpectTraceroute6(address string) []expect.Batcher {
+	return []expect.Batcher{
+		&expect.BSnd{S: "\n"},
+		&expect.BExp{R: "\\$ "},
+		&expect.BSnd{S: "traceroute -6 " + address + " -w1 > tr\n"},
```

Comment:
I personally find this more readable w/ `Sprintf`, but it's a personal preference, feel free to disregard.

Reply:
`Sprintf` felt noisy to me, this is why I used this version.
I will change it only if you insist.